egs/wsj/s5/conf/segment_long_utts.conf

```diff
@@ -0,0 +1,28 @@
+# TF-IDF similarity search options
```

Comment:
Vimal, this file is still here, and so is a similar file in the tedlium setup.  Did you not see my comment 5 days ago?